src/js/components/help/help.js

```diff
@@ -7,43 +7,26 @@ import { useTheme } from '@mui/material';
 import { getUserOrganization } from '../../actions/organizationActions';
 import { getDocsVersion, getIsEnterprise } from '../../selectors';
 import LeftNav from '../common/left-nav';
-import DeviceSupport from './device-support';
-import Devices from './devices';
-import GetStarted from './getting-started';
-import MoreHelp from './more-help-resources';
-import ReleasesArtifacts from './releases-and-artifacts';
-import BuildDemoArtifact from './releases-and-artifacts/build-demo-artifact';
 import Support from './support';
+import GetStarted from './getting-started';
+import MenderHub from './mender-hub';
 
 const components = {
   'get-started': {
-    title: 'Get started',
+    title: 'Getting started',
     component: GetStarted
   },
-  devices: {
-    title: 'Devices and device groups',
-    component: Devices
-  },
-  'device-and-os-support': {
-    title: 'Device and Operating System support',
-    component: DeviceSupport
-  },
-  'releases-artifacts': {
-    title: 'Releases and artifacts',
-    component: ReleasesArtifacts,
-    'build-demo-artifact': {
-      title: 'Building a demo application update Artifact',
-      component: BuildDemoArtifact
-    }
+  'support': {
+    title: 'Contact support',
+    component: Support
   },
-  support: {
-    title: 'Support',
-    component: Support,
-    hosted: true
+  'mender-hub': {
+    title: 'Mender Hub',
+    component: MenderHub
   },
-  'more-help-resources': {
-    title: 'More resources',
-    component: MoreHelp
+  'documentation': {
+    title: 'Documentation',
+    url: 'https://docs.mender.io'
```

Comment:
```suggestion
    url: 'https://docs.mender.io/'
```
might be what causes the link checker to fail...

Reply:
🤦 sorry, only now I see it... the docs link checker expect a string surrounded/ ending with the `` ` `` marks instead of the `'`... sorry about that!